### ISSUE TITLE
Stopped working in Chrome, no apparent errors

### ISSUE THREAD

**Issue by seanr** [unicorn-fail/dreditor#122]
No idea how to even debug this, but it just stopped working - it's not appearing on patches in issues, and I don't see any errors in the console except for this and it doesn't necessarily appear related to dreditor (it seems to happen on every page regardless of the site):

```
Denying load of chrome-extension://jifommjndpnefcfplgnbhabocomgdjjg/jquery-2.0.3.min.map. Resources must be listed in the web_accessible_resources manifest key in order to be loaded by pages outside the extension. 
```

Thi is Chrome 34.0.1847.131

**Comment by markhalliwell** [comment]
Are you logged in? Review buttons don't show up until you are.

**Comment by markhalliwell** [comment]
I don't think this is a Dreditor issue per se. It sounds like another extension is causing it to not load because of the error perhaps? Regardless, 1.2.6 is now out and I personally have verified this works correctly.

**Comment by frederickjh** [comment]
I am also having this problem in chrome. The review button does not show and tab completion does not work. Coloration of issues does work.

Using Chrome 37 on Ubuntu Linux.

See screenshot below with error console in Chrome for more info.

Frederick

![image](https://cloud.githubusercontent.com/assets/864159/4575473/a93d4812-4fa7-11e4-9d9f-6bfae2ad888b.png)

**Comment by frederickjh** [comment]
Dreditor website shows that the latest version is installed.
![image](https://cloud.githubusercontent.com/assets/864159/4575544/d45940c2-4fa8-11e4-981a-34e6cdc1299d.png)

**Comment by markhalliwell** [comment]
@frederickjh can you provide a screenshot of your Local Storage for a d.o page under the Resources tab of your inspector? That would help out a lot.

**Comment by frederickjh** [comment]
@markcarver dreditor is working again today. I just realized that yesterday I saw that the "Undo Recently Closed Tabs" extension was not adding anymore  tabs that were closed. So, I restarted Chrome. This morning I was on drupal.org and saw it was working.  The url for the  "Undo Recently Closed Tabs" extension is https://chrome.google.com/webstore/detail/jpcdannfionjoennkippohpibecglhic

If it does it again I will grab a screen shot a post it here, as I am guessing a screen shot when it is working of the Local Storage would not be any help.
